### PR TITLE
Added DSO - Deskside Support Officer

### PR DESCRIPTION
**Pull request by simonnebMOJ**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Ministry of Justice Acronyms
+![image](https://github.com/ministryofjustice/acronyms/assets/118196897/78e7f2f6-2703-4c1c-873b-32837b760686)# Ministry of Justice Acronyms
```
Comment (AntonyBishop) [inline README.md:1]: Is this Image supposed to be here as part of the PR. No reference to why it is added and for what purpose. Having seen the image I can't determine why it has been added?

**Review by AntonyBishop — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (simonnebMOJ) [reply README.md:1]: Oh that's an error and can be removed. Must have been a copy and paste issue

Comment (AntonyBishop) [reply README.md:1]: Thanks for confirming. Please can you remove it.

**Comment by simonnebMOJ** [comment]
ready for rechecking @AntonyBishop

**Review by AntonyBishop — APPROVED** [review]

**Comment by AntonyBishop** [comment]
Thank you

**Pull request merged**